docs/UserGuide.md

```diff
@@ -230,6 +230,29 @@ Expected behaviour upon failure:
 - Number does not correspond to any vendor: Displays error message "The number you have provided does not correspond to any vendor."
 - No input number: Displays error message "Please input an index"
 
+--------------------------------------------------------------------------------------------------------------------
+### Editing a vendor : `vendor edit`
+Edits the specified vendor in WedLog.
+
+```text
+vendor edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]...
```

Comment:
Not sure if this is an issue we should resolve in this PR or open a separate one, but the meaning of the ellipses in the input (here and elsewhere in the UG) are a bit unclear. I personally think we should just remove it and specify in words that users can input multiple tags. What do you think?